web/cypress.config.ts

```diff
@@ -15,5 +15,17 @@ export default defineConfig({
       framework: 'react',
       bundler: 'vite',
     },
+    setupNodeEvents(on, _config) {
+      on('before:browser:launch', (browser, launchOptions) => {
+        if (browser.family === 'chromium' && browser.name !== 'electron') {
+          launchOptions.preferences.default.intl = {
+            accept_languages: 'en-US,en',
+            selected_languages: 'en-US,en',
+          };
+
+          return launchOptions;
+        }
+      });
+    },
```

Comment:
This is just so the local setup uses the same as the CI everywhere. Was quite annoying when it picked Swedish instead of English all the time.